### PR TITLE
🌟add docker-compose.yaml container_name.

### PR DESCRIPTION
**Pull request by loks666**
# Description

When I was helping others deploy, I found that none of the services in Docker-compose do specify container_name, which results in docker automatically assigning a container name when Docker-Compose starts the container, which is usually "service name -1". This will misunderstand many people and cause them to misunderstand the problem

## Type of Change

Please delete options that are not relevant.
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement

# How Has This Been Tested?

run "docker-compose up -d && docker ps -a"
You can view the service name without a suffix

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] `optional` I have added tests that prove my fix is effective or that my feature works
- [x] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by crazywoola — CHANGES_REQUESTED** [review]
I would suggest using a prefix like `dify_api` to make sure it will not conflict with other named services.
And make sure you have changed all the yml files accordingly in the same folder.

**Review by takatost — CHANGES_REQUESTED** [review]
If you set `container_name`, it will prevent the service from setting multiple replicas, not recommended to do so;
If you want each container to have a custom prefix, you can deploy using `docker compose -p <your-project-name> up -d`.

**Comment by loks666** [comment]
This is also what puzzles me, why do other files have a specified container name, why not this file?

**Comment by takatost** [comment]
> This is also what puzzles me, why do other files have a specified container name, why not this file?

Are you referring to the Milvus Docker Compose file? Because its services are stateful and cannot be deployed multiple replicas, so you need to set the `container_name`.

**Comment by loks666** [comment]
Yes, it's docker-compose.milvus.yaml. Now I'm confused what do you mean by creating a copy? Is there a need to create multiple apis? Or is it docker swarm?

**Comment by bowenliang123** [comment]
No significant improvements in usability and clarity were found in this PR.
It never bother me to have `docker_web_1`, `docker_api_1` and `docker_worker_1` container names by default.

Or could you summarize the changes and impact with expected containers in lis for before and after this PR ?

**Comment by takatost** [comment]
I've closed this PR for now. Feel free to reach out if you have any questions.
